public/js/controllers/attackcost_controller.js

```diff
@@ -3,6 +3,7 @@ import TurboQuery from '../helpers/turbolinks_helper'
 import { getDefault } from '../helpers/module_helper'
 import globalEventBus from '../services/event_bus_service'
 import dompurify from 'dompurify'
+import _ from 'lodash-es'
```

Comment:
The pattern seems to be to specify what to import (`{ find }`), but I leave this PR to @buck54321 for a proper js review.